guides/source/action_controller_overview.md

```diff
@@ -463,13 +481,15 @@ class LoginsController < ApplicationController
 end
 ```
 
-To reset the entire session, use `reset_session`.
+To reset the entire session, use [`reset_session`][].
+
+[`reset_session`]: https://api.rubyonrails.org/classes/ActionController/Metal.html#method-i-reset_session
```

Comment:
Is it useful to link to the API documentation for methods that are public but have no description? It seems like we're offering the link as a way to see more about the method, but there's nothing to learn by clicking it.

![](https://user-images.githubusercontent.com/432189/98747800-3415eb00-23b0-11eb-9c06-76fcbf67fd00.gif)

Reply:
I think there are a few benefits to linking such methods:  Even when there is no method description, the API docs make it easy to see the source of the method, which can be useful.  Also, when searching for methods in the API docs, there can be multiple matches, so linking to the correct one could prevent confusion.  There is arguably also a UI / UX benefit from making API identifiers stand out a bit more in the guides, and a benefit from consistency.  And, of course, descriptions could be added for such methods in the future.  Calling attention to them might inspire people to contribute documentation... we can hope! :laughing:

Reply:
Yep, sounds good. I am thoroughly convinced!